src/ontology/uberon-edit.obo

```diff
@@ -224750,6 +224751,173 @@ is_a: UBERON:0012449 ! mechanoreceptor
 relationship: dc-contributor https://orcid.org/0000-0003-2473-2313
 property_value: dcterms-date "2024-06-12T13:51:05Z" xsd:string
 
+[Term]
+id: UBERON:8910000
+name: stomatogastric nervous system
+def: "The part of the nervous system that controls the stomach, such as in crabs, lobsters, and flies" [FlyBase:FBrf0089570, PMID:12966498, PMID:27450880, DOI:10.1016/B0-12-370878-8/00177-4]
+is_a: UBERON:0011216 ! organ system subdivision
+relationship: in_taxon NCBITaxon:6656 ! Arthropoda
+relationship: dc-contributor https://orcid.org/0000-0002-1909-7004
+
+[Term]
+id: UBERON:8910001
+name: stomatogastric ganglion
+def: "The stomatogastric ganglion (STG) consists of about 30 neurons that form two central pattern generator circuits in crustaceans. STG neurons have large soma (~50-100 um) and have complex branches. The crustacean stomach is a complex mechanical structure that grinds and filters foods. The STG contains the motor neurons that innervate the striated muscles that move the gastric mill and pyloric regions of the stomach." [PMID: 17009928] 
+synonym: "STG" EXACT [PMID: 17009928]
+is_a: UBERON:0011216 ! organ system subdivision
+relationship: part_of UBERON:8910000 ! stomatogastric nervous system
+relationship: dc-contributor https://orcid.org/0000-0002-1909-7004
+relationship: in_taxon NCBITaxon:6712 ! Astacidea
+relationship: in_taxon NCBITaxon:6752 ! Brachyura
+
+[Term]
+id: UBERON:8910010
+name: stomatogastric nerve (sensu Cancer borealis)
+def: "The stomatogastric nerve (SGN) is made up of four separate nerves that emerge from the paired commissural ganglia, the superior and inferior esophageal nerves (SONs and IONs)." [DOI:10.1016/B0-12-370878-8/00177-4]
+synonym: "SGN (sensu Cancer borealis)" EXACT [DOI:10.1016/B0-12-370878-8/00177-4]
+is_a: UBERON:0035014 ! functional part of brain 
+relationship: dc-contributor https://orcid.org/0000-0002-1909-7004
+relationship: in_taxon NCBITaxon:6712 ! Astacidea
+relationship: in_taxon NCBITaxon:6752 ! Brachyura
+relationship: part_of UBERON:8910000 ! stomatogastric nervous system
+
+[Term]
+id: UBERON:8910011
+name: dorsal gastric nerve (sensu Cancer borealis)
+def: "Nerve carrying dorsal gastric (DG) neurons which control the medial tooth movements in crustaceans." [PMID: 17009928]
+is_a: UBERON:0035014 ! functional part of brain
+relationship: dc-contributor https://orcid.org/0000-0002-1909-7004
+relationship: in_taxon NCBITaxon:6712 ! Astacidea
+relationship: in_taxon NCBITaxon:6752 ! Brachyura
+relationship: part_of UBERON:8910000 ! stomatogastric nervous system
+relationship: part_of UBERON:8910001 ! stomatogastric ganglion
```

Comment:
```suggestion
relationship: part_of UBERON:8910001 ! stomatogastric ganglion
```
Keep only one of the two axioms.
* Explanation:
If the `stomatogastric ganglion` is part of the `stomatogastric nervous system`, and another anatomical entity is part of the `stomatogastric ganglion`, then it is redundant to assert that the anatomical entity is also part of the `stomatogastric nervous system`. (The `part of` relationship is transitive.) Multiple classifications chains resulting from redundant assertions make the ontology difficult to edit and maintain in the long run.

Reply:
This is an interesting question we should unpack further. Many of these nerves are partially comprised of processes that originate from the stomatogastric ganglion, but then they extend far outside the ganglion around other parts of the stomatogastric nervous system. If you cut out the stomatogastric ganglion, many of these processes would still be there and would still be part of the stomatogastric nervous system.

Is there a different way we should capture the relationship?

This comment is similar for the other cases...

Reply:
> This is an interesting question we should unpack further. Many of these nerves are partially comprised of processes that originate from the stomatogastric ganglion, but then they extend far outside the ganglion around other parts of the stomatogastric nervous system. If you cut out the stomatogastric ganglion, many of these processes would still be there and would still be part of the stomatogastric nervous system.
> 
> Is there a different way we should capture the relationship?
> 
> This comment is similar for the other cases...

I see. There is the [RO:0002131 overlaps](https://www.ebi.ac.uk/ols4/ontologies/ro/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FRO_0002131?lang=en) relationship that might  capture this.